Product_table.txt

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE Product (
+    Id INT primary key not null,
+    Store_id INT not null,
+    Name char(256) not null,
+    Describe char(256),
+    Category_id INT,
+    Picture BYTEA,
+    Price INT not null,
+    Stock INT not null
```

Comment:
do you think INT is sql language
but PRIMARY KEY,  NOT NULL and CHAR  is not  sql language?

Reply:
you are right
I fix